Makefile

```diff
@@ -13,7 +13,7 @@ DATA_PATH ?= $(shell pwd)/data
 MJKEY_PATH ?= ~/.mujoco/mjkey.txt
 
 test:  ## Run the CI test suite
-test: RUN_CMD = nice -n 11 pytest -v -m 'not huge and not flaky' --durations=0
+test: RUN_CMD = nice -n 11 pytest -v -n 5 -m 'not huge and not flaky' --durations=0
```

Comment:
@gitanshu i did some experiments on my workstation and determined that `-n auto` is probably the best setting for this